src/biogeochem/CNProductsMod.F90

```diff
@@ -615,19 +615,12 @@ subroutine PartitionWoodFluxes(this, bounds, &
        if (pprod_tot > 0) then
           pprod10_frac = pprod10 / pprod_tot
           pprod100_frac = pprod100 / pprod_tot
-       else
-          ! Avoid divide by 0
-          pprod10_frac = 0._r8
-          pprod100_frac = 0._r8
+       else if (dwt_wood_product_gain_patch(p) > 0) then
+          call endrun(&
+               msg='ERROR: dwt_wood_product_gain_patch(p) > 0' // &
+               errMsg(sourcefile, __LINE__))
        end if
 
-       ! Note that the patch-level fluxes are expressed per unit gridcell area. So, to go
```

Comment:
This part needs to either stay here, or by @billsacks comment should be inside the if statement after line 617 above.